src/types/streams/streams_item.rs

```diff
@@ -15,7 +15,60 @@ pub struct StreamsItem {
     pub video_id: String,
     pub meta_transport_url: Url,
     pub stream_transport_url: Url,
+    pub state: Option<StreamItemState>,
     /// Modification time
     #[serde(rename = "_mtime")]
     pub mtime: DateTime<Utc>,
 }
+
+#[serde_as]
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct StreamItemState {
+    pub subtitle_track_id: Option<String>,
+    pub subtitle_language: Option<String>,
+    pub subtitle_delay_ms: Option<i64>,
```

Comment:
I would prefer the `subtitles_delay` and `audio_delay` to be documented to being milliseconds instead of adding it to the name + you can usually make a struct to specify that it's a delay of milliseconds if you want to be that strict.

It will be consistent with other parts of core where we have such values

Reply:
Do we have such a struct for miliseconds?

Reply:
No that's why I suggest just removing the MS from the field name